openpype/pipeline/colorspace.py

```diff
@@ -0,0 +1,116 @@
+import re
+import os
+import platform
+import PyOpenColorIO as ocio
+from openpype.settings import get_project_settings
+from openpype.pipeline import legacy_io
+from openpype.lib.log import Logger
+
+log = Logger.get_logger(__name__)
+
+
+def get_imagio_colorspace_from_filepath(
+    path, host=None, project_name=None, validate=True
+):
+    project_name = project_name or legacy_io.Session["AVALON_PROJECT"]
+    host = host or legacy_io.Session["AVALON_APP"]
```

Comment:
Rather use `host_name`. I would expect that `host` is an object of registered host.

Reply:
fixed in https://github.com/pypeclub/OpenPype/pull/4270